src/conspiracies/docprocessing/coref/coref_model.py

```diff
@@ -24,7 +25,10 @@ class CoreferenceModel(Predictor):
 
     Args:
         model_path(Union[Path, str, None], optional): Path to the model, if None, the
-            model will be downloaded to the default cache directory.
+            model will be downloaded according to default specifications of the language
+            parameter.
+        language(str): language of the (default) model to load. Has no effect if
```

Comment:
```suggestion
        language: language of the (default) model to load. Has no effect if
```
When using type hints I wouldn't recommend using types in the docstring (though I know that I have done it throughout the code base)